### PR TITLE
MGDDOC-60: Docs procedures for RHOAM

### PR DESCRIPTION
**Pull request by joaedwar**
# Description

Relates to the following Jiras
- https://issues.redhat.com/browse/MGDDOC-60
- https://issues.redhat.com/browse/MGDDOC-15

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist
- [x] This change requires a documentation update 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added a test case that will be used to verify my changes 
- [ ] Verified independently on a cluster by reviewer

### CODE REVIEW

**Comment by joaedwar** [comment]
@tonyxrmdavidson @trdoyle81, please review

**Comment by joaedwar** [comment]
@psturc Please review

File: test-cases/tests/documentation/i05-managing-users.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
```
Comment (psturc) [inline test-cases/tests/documentation/i05-managing-users.md:1]: is this a duplicate of `test-cases/tests/documentation/i04-managing-users.md`?

File: test-cases/tests/documentation/i03-adding-rhoam.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+## Description
+
+Procedure on adding RHOAM to your OSD cluster.
+
+- Jira ref link: https://issues.redhat.com/browse/MGDDOC-51
+
+- Pair contact: Joan Edwards
+
+* Doc link: hhttps://gitlab.cee.redhat.com/documentation-red-hat-managed-services/rhoam-documentation/-/edit/main/modules/getting-started/proc_adding-openshift-application-management-to-your-cluster.adoc
```
Comment (psturc) [inline test-cases/tests/documentation/i03-adding-rhoam.md:22]: typo
```suggestion
* Doc link: https://gitlab.cee.redhat.com/documentation-red-hat-managed-services/rhoam-documentation/-/edit/main/modules/getting-started/proc_adding-openshift-application-management-to-your-cluster.adoc
```

File: test-cases/tests/documentation/i04-accessing-rhoam.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  - manual-selection
+---
+
+# I04 - Accessing rhoam
+
+## Description
+
+Procedure on getting access to RHOAM
+
+- Jira ref link:
```
Comment (psturc) [inline test-cases/tests/documentation/i04-accessing-rhoam.md:18]: do we have jira link?

File: test-cases/tests/documentation/i05-managing-users.md (hunk 1 above)
Comment (joaedwar) [reply test-cases/tests/documentation/i05-managing-users.md:1]: nope. It is a different test cast.

File: test-cases/tests/documentation/i04-accessing-rhoam.md (hunk 3 above)
Comment (joaedwar) [reply test-cases/tests/documentation/i04-accessing-rhoam.md:18]: no Jira link/it is essentially apart of the previous jira on adding rhoam

**Comment by joaedwar** [comment]
@psturc Can we sync up to get these merge?

**Comment by joaedwar** [comment]
@psturc Not sure what the issue is. Can you have a look please?

**Comment by psturc** [comment]
@joaedwar run `npm run prettier` and commit the changes. Then it should be ok.

**Pull request merged**